Formula/progress.rb

```diff
@@ -18,14 +18,15 @@ def install
 
   test do
     pid = fork do
-      system "/bin/dd", "if=/dev/zero", "of=/dev/null", "bs=100000", "count=1000000"
+      system "/bin/dd", "if=/dev/urandom", "of=foo", "bs=512", "count=1048576"
     end
     sleep 1
     begin
-      assert_match(/dd/, shell_output("#{bin}/progress"))
+      assert_match "dd", shell_output("#{bin}/progress")
     ensure
       Process.kill 9, pid
       Process.wait pid
+      rm "foo"
```

Comment:
Frankly it's somewhat disturbing that upstream managed to rewrite that code such that `/dev/null` isn't sufficiently "real," but I digress …